app/controllers/user_activations_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class UserActivationsController < ApplicationController
+  include UsersHelper
+
+  def activate_form
+    assign_user(params[:id])
+    check_activation_permissions
+    redirect_to users_url if @user.active?
+  end
+
+  def deactivate_form
+    assign_user(params[:id])
+    check_activation_permissions
+    redirect_to users_url unless @user.active?
+  end
+
+  def activate
+    assign_user(params[:id])
+    check_activation_permissions
+    @user.activate! unless @user.active?
+    redirect_to users_url
+  end
+
+  def deactivate
```

Comment:
I do believe that we shouldn't break the CRUD convention, and instead have 2 controllers, one for activation and one for deactivations

Reply:
I think this is a hangover from how it was done in WEX. But I agree. Will sort this out now.

Reply:
@cintamani All fixed!